src/containers/Etherspot.tsx

```diff
@@ -78,9 +73,9 @@ const Etherspot = ({
       smartWalletOnly={smartWalletOnly}
     >
       <style>
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Regular.css');
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Medium.css');
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Bold.css');
+        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Regular.css'); @import
+        url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Medium.css'); @import
+        url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Bold.css');
```

Comment:
Previous one looks better readability wise right? It's subjective so you may ignore this

Reply:
you know I completely agree, idk why the linter felt the need to change this